sshd-common/src/test/java/org/apache/sshd/common/file/root/RootedFileSystemProviderTest.java

```diff
@@ -195,10 +219,15 @@ public void testChdir() throws IOException {
         }
     }
 
-    @Test(expected = NoSuchFileException.class)
+    @Test
     public void testChdirInvalid() throws IOException {
-        Path chdir = fileHelper.createDirectory(fileSystem.getPath(DOESNT_EXIST + getCurrentTestName()));
-        fail(String.format("Unexpected success in changing directory %s", chdir.toString()));
+        if (isTestingRealWindowsHostFS) {
```

Comment:
This looks identical to `testMkdirInvalid()` above...

Reply:
Yes please. Makes no sense.

Reply:
Deleted.